pages/index.tsx

```diff
@@ -271,6 +271,15 @@ function InstallSection() {
       />
     </div>
   );
+  const pacman = (
+    <div key="pacman" class="my-4 text-gray-700">
+      <p class="mb-2">Pacman (Arch Linux and Arch Based distros):</p>
```

Comment:
```suggestion
      <p class="mb-2">Pacman (Arch):</p>
```

Seems simpler like this.